cypress/commons/utils/apiUtils.js

```diff
@@ -149,6 +149,28 @@ const interceptLaunchScenario = (stubbingOptions) => {
   return alias;
 };
 
+const interceptStopScenarioRun = () => {
+  const alias = forgeAlias('reqStopScenarioRun');
+  cy.intercept({ method: 'POST', url: API_REGEX.STOP_SCENARIO_RUN, times: 1 }, (req) => {
+    if (stub.isEnabledFor('LAUNCH_SCENARIO')) {
+      const scenarioRun = req.url.match(API_REGEX.SCENARIO_RUN);
+      const scenarioRunId = scenarioRun[1];
+      const scenarioId = stub.getScenarioRunById(scenarioRunId);
+
+      const scenarioLastRun = {
+        id: scenarioRunId,
+        endTime: null,
+        message: null,
+        estimatedDuration: null,
+      };
+      stub.patchScenario(scenarioId, { state: 'Failed', lastRun: scenarioLastRun });
+
+      req.reply(scenarioRun);
```

Comment:
`req.reply` should be provided with the object of the stubbed _scenario run_ to return, but you're providing the result of the regex `match` here

Reply:
I would tell _scenarioLastRun_ instead of scenarioRun...